src/@utils/provider.ts

```diff
@@ -83,11 +84,11 @@ export async function getFileDidInfo(
 }
 
 export async function getFileUrlInfo(
```

Comment:
Same as over on ocean.js, if it does more than checking URLs, method needs a new name. Here we can simply remove `URL` and end up with `getFileInfo()`

Reply:
Done.